chart/templates/dag-processor/dag-processor-deployment.yaml

```diff
@@ -130,6 +130,9 @@ spec:
         {{- if .Values.dagProcessor.extraInitContainers }}
         {{- toYaml .Values.dagProcessor.extraInitContainers | nindent 8 }}
         {{- end }}
+{{- if .Values.dagProcessor.waitForMigrations.extraEnv }}
+{{ tpl (toYaml .Values.dagProcessor.waitForMigrations.extraEnv) $ | indent 12 }}
```

Comment:
Only waitForMigrations? What about the normal container?

Reply:
Wasn't aware if the dag-processor container interacted with the airflow db. Have added it for the main container as well